src/components/accordion/accordion.spec.tsx

```diff
@@ -530,21 +530,43 @@ describe("Accordion", () => {
       }
     );
 
-    it.each(Array.from({ length: 10 }).map((_, i) => 100 + i))(
-      "sets the width of the button to the value passed in via the buttonWidth prop",
-      (buttonWidth) => {
+    describe("sets the width of the button to the value passed in via the buttonWidth prop", () => {
+      it.each(["140px", "20%", "0.2"])("when passed as a %s string", () => {
+        const mockButtonWidth = "140px";
```

Comment:
I think we've accidentally missed a test parameter here :)
```suggestion
      it.each(["140px", "20%", "0.2"])("when passed as a %s string", (mockButtonWidth) => {
```